getVideo.js

```diff
@@ -0,0 +1,47 @@
+import Sequelize from 'sequelize';
+import Video from './db/models/video';
+import db from './config/db';
+
+export async function main(event, context, callback) {
+  // Do not wait for sequelize connection to close
+  context.callbackWaitsForEmptyEventLoop = false;
+
+  const headers = {
+    'Access-Control-Allow-Origin': '*',
+    'Access-Control-Allow-Credentials': true,
+  };
+
+  Video.init(db);
+
+  let response;
+  let statusCode;
+
+  const id = event.pathParameters.id;
+
+  try {
+    const result = await Video.findAll({ where: { "id": id } });
+    statusCode = 200;
+    let body = JSON.stringify(result[0]);
+
+    if (result.length === 0) {
+      statusCode = 204;
```

Comment:
Shouldn't it be a 404 - Not found?

Reply:
It's a bit of a discussion about it, the two top rated answers: https://stackoverflow.com/questions/11746894/what-is-the-proper-rest-response-code-for-a-valid-request-but-an-empty-data

404 would imply that an error/failure of the request occurred, broken link etc. 
This case is not really an error since it's a valid request that made it through successfully, though just returns empty data.  ?